core/src/main/scala/scalan/arrays/ArrayOps.scala

```diff
@@ -216,7 +216,7 @@ trait ArrayOps { self: Scalan =>
 trait ArrayOpsStd extends ArrayOps {
   self: ScalanStd =>
 
-  import TagImplicits.elemToClassTag
+  import TagImplicits.elemToClassTag  // do not remove, it's NOT unused
```

Comment:
Code analysis showed this import as unused, but build failed without it. A warning just in case.